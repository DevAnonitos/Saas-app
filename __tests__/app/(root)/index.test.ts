import Home from "@/app/(root)/page";
import CreatePodcast from "@/app/(root)/create-podcast/page";
import Discover from "@/app/(root)/discover/page";

describe("All Page Import", () => {
  it("should import All Pages", () => {
    expect(Home).toBeDefined();
    expect(CreatePodcast).toBeDefined();
    expect(Discover).toBeDefined();
  })

  it("should import All Pages", () => {
    expect(Home).toMatchSnapshot
    expect(CreatePodcast).toMatchSnapshot();
    expect(Discover).toMatchSnapshot();
  })
});