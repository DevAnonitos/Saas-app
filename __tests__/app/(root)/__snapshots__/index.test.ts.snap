// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All Page Import should import All Pages 1`] = `[Function]`;

exports[`All Page Import should import All Pages 2`] = `[Function]`;
